hdt_monitor/requirements/base.txt

```diff
@@ -1,9 +1,21 @@
-Django==1.9.4
+Django==2.2.18
 # Postgres
-psycopg2==2.8.4
+psycopg2-binary==2.8.6
 # Redis Cache
 redis==2.10.5
 django-redis-cache==1.6.5
 hiredis==0.2.0
-requests==2.9.1
-boto3==1.3.1
+requests==2.25.1
+boto3==1.17.7
+
+django-allow-cidr==0.3.1
+django-watchman==1.1.0
+
+splunk_handler==2.2.0
+python-json-logger==2.0.1
+
+git+ssh://git@github.com/Harvard-University-iCommons/django-icommons-common.git@v2.3#egg=django-icommons-common==2.3
+
+git+https://github.com/Harvard-University-iCommons/django-ssm-parameter-store.git@v0.3#egg=django-ssm-parameter-store==0.3
+
+django-debug-toolbar==3.2
```

Comment:
Any reason this was moved from local to base? It appears to still be only enabled for [local runs](https://github.com/penzance/hdt_monitor/pull/5/files#diff-31d8e8961fc018623a34c4fd7652861e44659421b458c8f556b1da682d44f77dR10), even if `DEBUG == True`.

Reply:
IIRC there was some reason that it was a pain to only have this conditionally installed; since we're building one image for all environments it's necessary to have it in there so we can activate the toolbar as needed via config.